src/mlir/verona-mlir.cc

```diff
@@ -54,8 +54,8 @@ namespace
   // Output file
   cl::opt<std::string> outputFile("o", cl::init(""), cl::desc("Output file"));
 
-  // Grammar file is not optional
-  std::string grammarFile;
+  // Grammar file
+  cl::opt<std::string> grammarFile("g", cl::init(""), cl::desc("Grammar file"));
```

Comment:
This change it not relevant

Reply:
Yep. It will go away once I rebase onto #347.

Reply:
Done.